modules/consensus/consistency.go

```diff
@@ -112,9 +112,9 @@ func (cs *ConsensusSet) checkSiacoins() error {
 // checkSiafunds counts the siafund outputs and checks that there are 10,000.
 func (cs *ConsensusSet) checkSiafunds() error {
 	totalSiafunds := types.ZeroCurrency
-	for _, sfo := range cs.siafundOutputs {
+	cs.db.forEachSiafundOutputs(func(sfoid types.SiafundOutputID, sfo types.SiafundOutput) {
 		totalSiafunds = totalSiafunds.Add(sfo.Value)
-	}
```

Comment:
this forEach on the other hand makes more sense. You are adding to totalSiafunds repeatedly.